_literate/EX-wine/Project.toml

```diff
@@ -6,5 +6,4 @@ MLJ = "add582a8-e3ab-11e8-2d5e-e98b27df1bc7"
 MLJLinearModels = "6ee0df7b-362f-4a72-a706-9e79364fb692"
 MLJMultivariateStatsInterface = "1b6a4a23-ba22-4f51-9698-8599985d3728"
 NearestNeighborModels = "636a865e-7cf4-491e-846c-de09b730eb36"
-PyPlot = "d330b81b-6aea-500a-939a-2ce795aea3ee"
```

Comment:
add `Plots` here?

Reply:
Of course, it worked like magic without it back then so I missed that.